keel-clouddriver/keel-clouddriver.gradle

```diff
@@ -7,8 +7,13 @@ dependencies {
   implementation "com.fasterxml.jackson.core:jackson-databind"
   implementation "com.fasterxml.jackson.core:jackson-annotations"
   implementation "com.fasterxml.jackson.module:jackson-module-kotlin"
+  implementation "com.github.ben-manes.caffeine:caffeine"
   implementation "com.github.jonpeterson:jackson-module-model-versioning:1.2.2"
   implementation "com.fasterxml.jackson.datatype:jackson-datatype-jsr310"
+  implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:$coroutinesVersion"
+  implementation "org.slf4j:slf4j-api"
```

Comment:
slf4j needs to be explicitly declared now in a bunch of spots...

we could theoretically add this as an implementation dependency in the `subprojects` block in build.gradle (though that only right now has the bom imports so this would potentially add to the dependency graph if something didn't actually do any logging...)

Reply:
I think even if something doesn't do any logging you generally want the API available when making changes to code.